### PR TITLE
feat(stdlib): add and use lv_malloc_zeroed

### PR DESCRIPTION
**Pull request by niklasf**
`lv_malloc()` followed by `lv_memzero()` is a very common pattern. We could introduce `lv_calloc()` for brevity and to use:

* `calloc()` for `LV_STDLIB_CLIB`
* `m_malloc0()` for MicroPython

### CODE REVIEW

File: src/stdlib/lv_mem.c [hunk 1]
```diff
@@ -83,8 +83,8 @@ void * lv_malloc(size_t size)
         LV_LOG_INFO("used: %6d (%3d %%), frag: %3d %%, biggest free: %6d",
                     (int)(mon.total_size - mon.free_size), mon.used_pct, mon.frag_pct,
                     (int)mon.free_biggest_size);
-        return NULL;
 #endif
+        return NULL;
```
Comment (niklasf) [inline src/stdlib/lv_mem.c:87]: Notable change

**Comment by kisvegabor** [comment]
Do you know if `calloc` internally can be more efficient than `malloc` + `memset`?

**Comment by niklasf** [comment]
It depends. Compilers generally optimize `malloc` + `memset` to `calloc`, already. So we would only see an effect for previously missed optimizations. And then it only helps if the allocator has prepared some zeroed memory.

It's definitely convenient, though, especially if `size` is computed.

**Comment by kisvegabor** [comment]
Sorry, but I don't feel it strong enough to have new API for this. Maybe just a helper like 
```c
void * lv_malloc_zero(size_t size)
{
     void * p =lv_malloc(size);
     if(p) lv_memzro(p, size);
    return p;
}
```

I think it's better because
1. Its API is the same as malloc
2. It doesn't require a new function in all stdlib wrappers

**Comment by niklasf** [comment]
Sure. Seems like a useful utility to add even in this way.

**Comment by kisvegabor** [comment]
Great, thank you!

**Pull request merged**